<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>676</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">gridline-color: rgb(85, 85, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>10000</height>
    </size>
   </property>
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(162, 167, 255);</string>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(162, 167, 255);</string>
    </property>
    <property name="text">
     <string>Soumili's Sudoku Solver :)</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="sudokuTable">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(162, 167, 255);
font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>9</number>
    </property>
    <property name="columnCount">
     <number>9</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>48</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>62</number>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>38</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>45</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(162, 167, 255);</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showRecursionButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(162, 167, 255);</string>
    </property>
    <property name="text">
     <string>Show Recursion</string>
    </property>
   </widget>
   <widget class="QDial" name="speedDial">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(162, 167, 255);
font: 700 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Recursion speed </string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRecursionButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(162, 167, 255);</string>
    </property>
    <property name="text">
     <string>Stop Recursion</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
